### PR TITLE
Sp3 ban user

### PR DESCRIPTION
**Pull request by Spellbreeze**
Collaborator: Xander

### CODE REVIEW

**Comment by Spellbreeze** [comment]
Looks like this includes auth fixes. Should be included after sp3_tokenValidation, sorry.

File: backend/controllers/auth_controller.js [hunk 1]
```diff
@@ -2,14 +2,15 @@ import jwt from 'jsonwebtoken'
 
 import User from '../models/users_model.js'
 
+// This function searches for a user with the matching token
 const auth = async (req, res, next) => {
-  console.log('Request Body')
-  console.log(req.body)
```
Comment (madhura112) [inline backend/controllers/auth_controller.js]: good that we removed unnecessary console.logs

**Review by madhura112 — APPROVED** [review]

File: backend/controllers/users_controller.js [hunk 2]
```diff
@@ -24,6 +24,8 @@ export const loginUser = asyncHandler(async (req, res) => {
     const user = await User.findByCredentials(email, password)
     if (!user) {
       return res.status(401).send({ error: 'Login failed! Check authentication credentials' })
+    } else if (user.isBanned) {
+      return res.status(402).send({ error: 'Login failed! User is banned' })
```
Comment (parkaransingh) [inline backend/controllers/users_controller.js:28]: We need to test this on the front end next

**Review by parkaransingh — APPROVED** [review]
LGTM.

**Pull request merged**